server/routes/add-steps/locale.json

```diff
@@ -0,0 +1,39 @@
+{
+  "en": {
+    "page": {
+      "title": "Add steps to goal - {{ applicationName }}",
+      "errorTitle": "Error: Add steps to goal - {{ applicationName }}"
+    },
+    "mainHeading": {
+      "title": "Add steps to {{ subject.givenName }} goal",
+      "caption": "{{ goal.title }}"
+    },
+    "detailHeading": "View information from John’s assessment",
+    "step": {
+      "description": {
+        "label": "What should they do to achieve the goal?",
+        "hint": "Search for a suggested step or enter your own. Add one step at a time."
+      },
+      "actor": {
+        "label": "Who will do this step?",
+        "hint": "Add one person or agency.",
+        "options": [
+          "{{ subject.givenName }}",
+          "Probation practitioner",
+          "Programme staff",
+          "Partnership agency",
+          "Commissioned rehabilitative services (CRS) provider",
+          "Someone else (include who in the step)"
+        ]
+      }
+    },
+    "saveButtonText": "Save and continue",
+    "addAnotherButton": "Add another step",
+    "removeStepLinkButton": "Remove",
+    "errors": {
+      "step-description": {
+        "isNotEmpty": "You need to enter a step"
```

Comment:
`Select or enter what they should do to achieve the goal` is the text given on the errors spreadsheet.